scalafix-tests/input/src/main/scala/test/explicitResultTypes/Rename.scala

```diff
@@ -3,11 +3,11 @@ rules = ExplicitResultTypes
 */
 package test.explicitResultTypes
 
-import scala.collection.{Seq => SSeq}
+import scala.collection.immutable.{List => LList}
```

Comment:
why this change?

Reply:
scala.Seq is now an alias for scala.collection.immutable.Seq[A] (instead of scala.collection.Seq[A]) in 2.13.
So I would need different output tests. The goal is to test the Rename ... this is why I did this